src/Task10/task10.php

```diff
@@ -0,0 +1,13 @@
+<?php
+$word = "China";
+
+function changeChr (string $st)   string
```

Comment:
Check your style.
Use colon after function declaration for return type hinting.